packages/sdk/src/types/LayoutTypes.ts

```diff
@@ -77,6 +77,17 @@ export type BleedDeltaUpdate = {
     areBleedValuesCombined?: boolean;
 };
 
+export type LayoutPreset = {
+    name: string;
+    intent: LayoutIntent;
+    unit: MeasurementUnit;
+    width: string;
+    height: string;
+    duration?: number;
+    bleed?: LayoutBleed;
```

Comment:
On engine side, this variant is used. Which one is correct @NicolaVerbeeck ?

```suggestion
    bleed?: BleedDeltaUpdate;
```